actions/scripts/oc_jenkins.sh

```diff
@@ -57,44 +58,27 @@ function test_all_actions() # $1 = card, $2 = accel
 
 	RC=0;
 	# Get SNAP Action number from Card
-	MY_ACTION=`./software/tools/snap_maint -C $card -m 1`
+
+	# MY_ACTION=`./software/tools/oc_maint -C $card -m 1`
+	# oc_maint -m1 not working for the moment ==> Using the following as a workaround
+	MY_ACTION=`./software/tools/oc_maint -C $card -v | grep -A10 -e "-+-" | grep -ve "-+-" | awk '{print $2}'`
+
 	for action in $MY_ACTION ; do
 		run_test=1;
 		case $action in
-		*"10140000")
-			test_10140000 $card $accel
+		*"10142000")
+			test_10142000 $card $accel
```

Comment:
Why hdl_example needs a special function "test_10142000"? I see other places just calling the test script.